net/test/ut/commands.c

```diff
@@ -669,7 +669,8 @@ static void commands_node_thread2(struct net_test_cmd_node *node)
 		for (i = 1; i < ARRAY_SIZE(nodes); ++i) {
 			strncat(buf, nodes[i].ntcn_addr, NTC_ADDR_LEN_MAX - 1);
 			if (i != ARRAY_SIZE(nodes) - 1)
-				strncat(buf, ",", 1);
+				//strncat(buf, ",", 1); //Nalini
+				strcat(buf, ",");
```

Comment:
Why?
What is the reason for such change?

Reply:
there was some compiler message, because of buff array size, that is why i have increased buff to  64 bytes in size, and replaced strncat with strcat only. but the commented code was original code. That's why I kept there by tagging my name. which i will remove later after testing. both code are equivalent ,however